### ISSUE TITLE
MSYS2 compilation error in dateentry.cc

### ISSUE THREAD

**Issue by agriggio** [artpixls/ART#104]
**[Original report](https://bitbucket.org/agriggio/art/issues/104) by Gaaned92 (Bitbucket: [Gaaned92](https://bitbucket.org/Gaaned92), GitHub: [Gaaned92](https://github.com/Gaaned92)).**

----------------------------------------

W10, MSYS2, ART 1.3-117-gd8f83c5c8

```c++
D:/ART/ARTSOURCE/ART/rtgui/dateentry.cc: In member function 'void DateEntry::on_enter()':
D:/ART/ARTSOURCE/ART/rtgui/dateentry.cc:124:17: error: 'strptime' was not declared in this scope; did you mean 'strftime'?
  124 |     char *end = strptime(val.c_str(), fmt.c_str(), &t);
      |                 ^~~~~~~~
      |                 strftime
```

‌

**Comment by agriggio** [comment]
**Original comment by Carmelo DrRaw (Bitbucket: [Carmelo DrRaw](https://bitbucket.org/Carmelo DrRaw), ).**

----------------------------------------

I confirm the issue also with mingw under a cross-compilation environment. Build log can be found here: [https://api.travis-ci.org/v3/job/695029619/log.txt](https://api.travis-ci.org/v3/job/695029619/log.txt)

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

thanks, I'll fix later today or tomorrow

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

use std::get_time instead of strptime

Fixes #104

**Comment by agriggio** [comment]
**Original comment by Carmelo DrRaw (Bitbucket: [Carmelo DrRaw](https://bitbucket.org/Carmelo DrRaw), ).**

----------------------------------------

Fix confirmed: [https://travis-ci.org/github/aferrero2707/art-win64/jobs/695029619](https://travis-ci.org/github/aferrero2707/art-win64/jobs/695029619)
